wiki/Registration/it.md

```diff
@@ -1,55 +1,22 @@
----
-outdated: true
----
-Come registrarsi?
------------------
+# Registrazione
 
-Registrare un account in osu! è semplice come contare fino a tre! Segui le istruzioni qui sotto.
+*Attenzione: Il possesso di account multipli è in ogni momento una violazione delle [regole di osu!](/wiki/Rules)*
```

Comment:
Can you rephrase this to make sure `osu!` isn't in the end. This is an exclamatory sentence and should end with a `!`, and should avoid `osu!!` if possible.